usr/share/rear/restore/NBU/default/400_restore_with_nbu.sh

```diff
@@ -18,7 +18,8 @@ fi
 LogPrint "RUN: /usr/openv/netbackup/bin/bprestore $bprestore_args"
 LogPrint "Restore progress: see $TMP_DIR/bplog.restore"
 LANG=C /usr/openv/netbackup/bin/bprestore $bprestore_args
-if (( $? > 1 )) ; then
-    Error "bprestore failed (return code = $?)"
+rc=$?
+if (( $rc > 1 )) ; then
```

Comment:
@rmetrich 
do you perhaps know why in case of NBU bprestore a return code '1' is not an error?
If yes, could you add a comment that explains why here return code '1' means "o.k."
cf. what we did for TSM, see
https://github.com/rear/rear/pull/1643#issuecomment-351366255
that lead to commits like
https://github.com/rear/rear/pull/1645/commits/96c80a4a879a4f62beecdd044e4a46dfd97adfcd
and
https://github.com/rear/rear/pull/1643/commits/d7d9b3c91bb626f18bd3fdfd9a551d864dc0bad1

Reply:
Don't know why, I just saw the broken error status from a customer log